JedisManager.cfc

```diff
@@ -123,6 +123,37 @@ component {
         }
     }
 
+     /**
+     * Delete the given key and associated value from cache.
```

Comment:
In a future version of this lib you could figure out which server version ACF is running and have an internal switch to use DEL or UNLINK depending on ACF server and Jedis lib included.

I'd create a separate ticket for that though.